examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -28,62 +28,50 @@ using JLD2
 using Oceananigans
 using Oceananigans.Units: minute, minutes, hour
 
-# ## A vertically-stretched grid
+# ## The grid
 #
-# We use 32³ grid points with 2 meter grid spacing in the horizontal and
+# We use 32³ grid points with 2 m grid spacing in the horizontal and
 # varying spacing in the vertical, with higher resolution closer to the
-# surface. We use a two-parameter generating function to specify the
-# vertical cell interfaces:
+# surface,
 
-Nz = 24 # number of points in the vertical direction
-Lz = 32 # domain depth
-refinement = 1.2 # controls spacing near surface (higher means finer spaced)
-stretching = 8   # controls rate of stretching at bottom 
+σ = 1.1 # stretching factor
+Nz = 24
+Lz = 32
 
-## Normalized height ranging from 0 to 1
-h(k) = (k - 1) / Nz
-
-## Linear near-surface generator
-ζ₀(k) = 1 + (h(k) - 1) / refinement
-
-## Bottom-intensified stretching function 
-Σ(k) = (1 - exp(-stretching * h(k))) / (1 - exp(-stretching))
-
-## Generating function
-z_faces(k) = Lz * (ζ₀(k) * Σ(k) - 1)
+hyperbolically_spaced_faces(k) = - Lz * (1 - tanh(σ * (k - 1) / Nz) / tanh(σ))
 
 grid = VerticallyStretchedRectilinearGrid(size = (32, 32, Nz), 
                                           x = (0, 64),
                                           y = (0, 64),
-                                          z_faces = z_faces)
+                                          z_faces = hyperbolically_spaced_faces)
 
 # We plot vertical spacing versus depth to inspect the prescribed grid stretching:
 
 plot(grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz],
-     marker = :circle,
-     ylabel = "Depth (m)",
-     xlabel = "Vertical spacing (m)",
-     legend = nothing)
+      marker = :circle,
+      ylabel = "Depth (m)",
+      xlabel = "Vertical spacing (m)",
+      legend = nothing)
 
 # ## Buoyancy that depends on temperature and salinity
 #
 # We use the `SeawaterBuoyancy` model with a linear equation of state,
 
 buoyancy = SeawaterBuoyancy(equation_of_state=LinearEquationOfState(α=2e-4, β=8e-4))
 
-# where ``α`` and ``β`` are the thermal expansion and haline contraction
+# where $α$ and $β$ are the thermal expansion and haline contraction
 # coefficients for temperature and salinity.
 #
 # ## Boundary conditions
 #
 # We calculate the surface temperature flux associated with surface heating of
-# 200 W m⁻², reference density `ρₒ`, and heat capacity `cᴾ`,
+# 200 W m⁻², reference density `ρ`, and heat capacity `cᴾ`,
```

Comment:
if we'll call this `ρ` then the variable below should be ρ instead ρₒ?

Reply:
word. I don't know how these changes snuck in btw.

Reply:
```suggestion
# 200 W m⁻², reference density `ρₒ`, and heat capacity `cᴾ`,
```